### PR TITLE
New  registry entry for 'Guatemala Tax Identification Number' (GT-NIT)

### PR DESCRIPTION
**Pull request by duncandewhurst**
A new list has been proposed with the code GT-NIT

**List title:** Guatemala Tax Identification Number

add draft GT-NIT list

 Preview the platform with this list at [http://org-id.guide/_preview_branch/GT-NIT](http://org-id.guide/_preview_branch/GT-NIT)  (visiting [http://org-id.guide/list/GT-NIT](http://org-id.guide/list/GT-NIT) when the preview is active or check the files changed options above.

Some further information is required before the list can be merged.

### CODE REVIEW

File: lists/gt/gt-nit.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "GT"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "trust"
+  ],
+  "sector": [],
+  "code": "GT-NIT",
+  "confirmed": false,
```
Comment (siwhitehouse) [inline lists/gt/gt-nit.json]: I think you need to set confirmed to 'true' or the list won't appear in org-id

**Comment by kd-ods** [comment]
@duncandewhurst - how's this PR coming. Anything I or @siwhitehouse can help with?

File: lists/gt/gt-nit.json (hunk 1 above)
Comment (duncandewhurst) [reply lists/gt/gt-nit.json]: Thanks! Fixed in https://github.com/org-id/register/pull/382/commits/ddb67c2c2a610baf625656cae5ee436a12618eb9

**Comment by duncandewhurst** [comment]
@kd-ods @siwhitehouse I think this can be merged now. I don't know whether the list type is primary or secondary, but we can update that in the future if more information becomes available

**Review by siwhitehouse — APPROVED** [review]
This looks good to me, thanks @duncandewhurst. I'd not spotted the lack of list type, but I'd be happy if you wanted to add this as a primary list on the basis that it is possible to request a NIT through the site using the "Solicitar NIT" page(s)

**Comment by duncandewhurst** [comment]
As it's a tax ID it's not clear to me whether the same legal entity can have multiple IDs, e.g. if it registers as a tax payer, then deregisters and later reregisters with a new ID, so better not to specify the type for now, I think

**Comment by siwhitehouse** [comment]
I've run the web hook to import this list into the org-id.guide website. The list has been given a score of 0% and I am assuming that this is because it doesn't have a listType. I'm going to retract on what I said before as I've read the [listType entry in the schema](https://github.com/org-id/register/blob/master/schema/codelist-listType.json), which includes

```
"code":"secondary",
            "title":{
                "en":"Secondary"
            },
            "description":{
                "en":"This is list managed by an official source. Examples of secondary lists are VAT or Tax identifiers, or employer identification numbers.  Not all organisations of a given legal type will have these identifiers (for example, if they don't pay tax for a tax ID), and it may be possible to a single organisation to have more than one identifier from this list, or it's identifier to change at certain points in time. Being on this list relies upon some status the organisation has, or some official transactions it has engaged in."
            },
            "quality_score":50
        },
```

On that basis I think we should add 

`"listType": "secondary" `

to this entry.

**Pull request merged**